packages/flutter/lib/src/widgets/dismissable.dart

```diff
@@ -38,16 +38,25 @@ enum DismissDirection {
   down
 }
 
-/// Can be dismissed by dragging in one or more directions.
+/// Can be dismissed by dragging in the indicated [direction].
 ///
-/// The child is draggable in the indicated direction(s). When released (or
-/// flung), the child disappears off the edge and the dismissable widget
+/// Dragging or flinging this widget in the [DismissDirection] causes the child
+/// to slide out of view. Following the slide animation, the Dismissable widget
 /// animates its height (or width, whichever is perpendicular to the dismiss
 /// direction) to zero.
+///
+/// Backgrounds can be used to implement the "leave-behind" idiom. If a background
```

Comment:
We could do a pass over the entire code base to link constants, classes, and etc to the MD spec.  There's a bunch of work there.